include/builtins.h

```diff
@@ -6,13 +6,14 @@
 /*   By: lporoshi <lporoshi@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/12/09 17:31:39 by lporoshi          #+#    #+#             */
-/*   Updated: 2023/12/10 14:44:50 by lporoshi         ###   ########.fr       */
+/*   Updated: 2023/12/10 17:51:13 by lporoshi         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #ifndef BUILTINS_H
 # define BUILTINS_H
 
 int	echo(char **argv, bool no_newline_flag);
```

Comment:
I'd suggest to name all builtins like "builtin_echo", "builtin_pwd" or something similar